src/lib/modules/response/resp_delay.go

```diff
@@ -22,9 +21,11 @@ const (
 // in a Route struct and supplies it to a function in turn to handle it accordingly. We create
 // channels to run tests for each route in parallel, speeding up the process
 func HandleResponseDelayForRoute(responseChains []*tsdb.Chain, route parser.Routes, tsdbNameHash string, wg *sync.WaitGroup) {
-	routeSuffix := filters.RouteDestroyer(route.URL + "_" + route.Route)
+	// routeSuffix := filters.RouteDestroyer(route.URL + "_" + route.Route)
```

Comment:
clean the code, and change the naming conventions, do not keep it as a hash, if its not a hash value.

Reply:
@ganeshpatro321 Yeah! I will be modifying the above 2 parts.